index.js

```diff
@@ -1,19 +1,157 @@
+    img: '/images/placeholder.png',
+    technologies: ['css', 'html', 'javascript', 'Ruby'],
+    linkDemo: '',
+    sourceCode: 'https://github.com/til2to/porfolio_projects/tree/main',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    img: '/images/placeholder.png',
+    technologies: ['css', 'html', 'javascript', 'Ruby'],
+    linkDemo: '',
+    sourceCode: 'https://github.com/til2to/porfolio_projects/tree/main',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    img: '/images/placeholder.png',
+    technologies: ['css', 'html', 'javascript', 'Ruby'],
+    linkDemo: '',
+    sourceCode: 'https://github.com/til2to/porfolio_projects/tree/main',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    img: '/images/placeholder.png',
+    technologies: ['css', 'html', 'javascript', 'Ruby'],
+    linkDemo: '',
+    sourceCode: 'https://github.com/til2to/porfolio_projects/tree/main',
+  },
+];
+
+projectArray.forEach((project, index) => {
+  let tech = '';
+  project.technologies.forEach((item) => {
+    tech += `<li class="tech-item">
+    <span>${item}</span>
+    </li>`;
+  });
+
+  const projectItem = `<section class="ellipse_19">
+    <figure class="img_placeholder">
+      <img class="imagePlaceholder" src=${project.img} alt="project image">
+    </figure>
+
+    <section class="post_info">
+      <article>
+        <h2 class="title_post">${project.name}</h2>
+      </article>
+      
+      <article>
+        <p>
+          ${project.description}
+        </p>
+      </article>
+
+      <article class="tags">
+        <ul>
+          ${tech}
+        </ul>
+      </article>
+
+      <button class="button" id=${index}>See Projects</button>
+    </section>
+  </section>`;
+
+  projectContainer.innerHTML += projectItem;
+});
+
+const projectBtns = document.querySelectorAll('.button');
+
+const modalBtn = document.querySelector('#modal-close');
+
+modalBtn.addEventListener('click', () => {
+  modalMainContainer.style.display = 'none';
+  blurGround.style.filter = 'blur(0)';
+  blurGround.style.opacity = '1';
+});
+
+const modalContent = (project) => {
+  // blurGround.style.filter = 'blur(2px)';
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻